packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/About.2.doc.mdx

```diff
@@ -14,14 +12,102 @@ import WorklistDataGridColumnStates from './Examples.3/WorklistDataGridColumnSta
+* [Design Decisions](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#design-decisions)
+* [Examples](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#examples)
+* [Accessibility](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#accessibility)
+* [Implementation](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#implementation)
+* [Terra Standards](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#terra-standards)
+* [Properties](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#properties)
+
+## Overview
+  The WorklistDataGrid is an [accessible grid](https://www.w3.org/WAI/ARIA/apg/patterns/grid/) component that allows the user to display data in a table like structure with rows and columns. The grid provides core functionality that a consumer would need while allowing flexibility for the consumers to control specific behavior of the grid based on their requirements.
+
+  The WorklistData has a header row and one or more data rows. The columns may allow sorting based on the consumers specification.
+  At any given point, only one cell will be focusable and tabbing into the Worklist DataGrid will focus that cell. The arrow keys are used for navigation. See [keyboard interactions](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#keyboard-interactions) for all supported keys.
+
+  ### Features
+
+  * [Navigation](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/default-worklist-data-grid)
+  * [Masked Cells](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/default-worklist-data-grid)
+  * [Cell selection](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/cell-selection)
+  * [Row selection](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/row-selection)
+  * [Pinned columns](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/pinned-columns)
+  * [Column Sorting](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/worklist-data-grid-column-states)
+  * [Column Error States](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/worklist-data-grid-column-states)
+  * [Single Cell Navigation (Dive-In)](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/worklist-data-grid-focusable-cell)
+
+Refer to the [examples](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#examples) section for detailed demonstration of the features supported by the WorklistDataGrid.
 
-## Getting Started
+## Design Decisions
+For the most part, the Worklist DataGrid supports the [grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/) as documented by the ARIA Authoring Practices Guide.
 
+The Worklist DataGrid:
```

Comment:
Data grid is two words: https://en.wikipedia.org/wiki/Data_grid.
Since this is also a proper noun, it'll be `Worklist Data Grid`.